index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <ul>
+            <li> <p class="welcome">Welcome</p></li>
+            <li> <img src="img/union.png" alt="menu bar"> </li>
```

Comment:
- [ ] The hamburger image that showed on the page is the the expected one. It is expected to be wrapped inside a `nav` tag. This is for for semantics reason, and the right position for convention and user experience for mobile devices. Kindly look into it and fix.

### Your Design
![your-hamburger](https://user-images.githubusercontent.com/26597432/204791154-9ba97ce9-9e74-499c-bbaa-5053922ff7a2.png)

### Temp Design
![temp-hamburger](https://user-images.githubusercontent.com/26597432/204791172-4ad332df-e0ac-4be5-88bf-5e69d776074d.png)

Reply:
is it proper to rename the header  section nav section, since the design do not have a header?
please is this the right way to go about it?

Reply:
![image](https://user-images.githubusercontent.com/105445428/204796659-bb3562fd-0117-4f9c-9e0f-3a794f822de5.png)
the hamburger logo is showing on my system. How can I make it to be showing o reviwer's system